packages/manager/apps/web/client/app/dns-zone/history/dashboard/view/view.controller.js

```diff
@@ -0,0 +1,21 @@
+export default class ViewDnsZoneController {
+  constructor($state) {
+    this.$state = $state;
+  }
+
+  $onInit() {
+    this.dnsZoneData = null;
+    this.loadingDnsZoneData = true;
+
+    this.getDnsZoneData(this.url)
+      .then((res) => {
+        this.dnsZoneData = res;
+      })
+      .catch(() => {
+        this.$state.go('^');
```

Comment:
usually, it's the `goBack` function, so you can use it instead of recreating a new routing call